### PR TITLE
Add Intel's PresentMon to third_party

### PR DESCRIPTION
**Pull request by pierricgimmig**
"PresentMon is a tool to capture and analyze ETW events related to swap
chain presentation on Windows. It can be used to trace key performance
metrics for graphics applications (e.g., CPU and Display frame durations
and latencies) and works across different graphics APIs, different
hardware configurations, and for both desktop and UWP applications."

Snapshot of commit 90c5b0530723a9b56c543be7d7e537951c25809c
Mon Jun 13 09:54:38 2022 -0700

https://github.com/GameTechDev/PresentMon

### CODE REVIEW

File: cmake/FindPresentMon.cmake [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+add_library(PresentMon INTERFACE)
+
+set(DIR third_party/PresentMon)
+target_include_directories(PresentMon SYSTEM INTERFACE ${DIR})
+
+add_library(PresentMon::PresentMon ALIAS PresentMon)
+
+
+# # Copyright (c) 2021 The Orbit Authors. All rights reserved.
```
Comment (danielfenner) [inline cmake/FindPresentMon.cmake:13]: The lines below look as if they are left over from debugging, right?

File: cmake/FindPresentMon.cmake [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
```
Comment (beckerhe) [inline cmake/FindPresentMon.cmake:1]: Nit: 2022

**Review by beckerhe — APPROVED** [review]
What Daniel said. Otherwise LGTM

File: cmake/FindPresentMon.cmake (hunk 1 above)
Comment (pierricgimmig) [reply cmake/FindPresentMon.cmake]: Argh, yes, thanks!

**Pull request merged**